docs/en/operation/Troubleshooting.md

```diff
@@ -218,12 +227,12 @@ Typical issues:
 ### Q: I'm trying to deploy Alluxio in a cluster with Spark and HDFS. Are there any suggestions?
 
 A: Please follow [Running-Alluxio-on-a-Cluster]({{ '/en/deploy/Running-Alluxio-On-a-Cluster.html' | relativize_url }}),
-[Configuring-Alluxio-with-HDFS]({{ '/en/ufs/HDFS.html' | relativize_url }}).
+[Configuring-Alluxio-with-HDFS]({{ '/en/ufs/HDFS.html' | relativize_url }}),
+and [Configuring-Spark-with-Alluxio]({{ '/en/compute/Spark.html' | relativize_url }}).
 
 Tips:
 
 - The best performance gains occur when Alluxio workers are co-located with the nodes of the computation frameworks.
-- You can use Mesos and Yarn integration if you are already using Mesos or Yarn to manage your cluster.
```

Comment:
Any reason for us to delete this?

Reply:
I don't think Mesos or Yarn integration are still there, are they? @apc999